include/zos-tls.h

```diff
@@ -37,4 +37,39 @@ template <typename T> class __Z_EXPORT __tlssim {
 };
 
 #endif // __cplusplus
+
+/* 
+These macros provide a workaround for the non-existing thread local storage (TLS) 
+support for C applications on z/os. It leverages the pthread_key_create, pthread_getspecific 
+and pthread_setspecific calls to acheve TLS.
+*/
+
+static void tls_destructor(void *value) {
+    free(value);
+}
+
+#define GET_KEY_VAL(key_name, key_status, val, size) \
+   if(key_status == 0){\
```

Comment:
```suggestion
   if (key_status == 0) {\
```

Reply:
Also please add a blank space before the continuation `\` for all continued lines so they're consistent.